config/services.yaml

```diff
@@ -32,6 +32,7 @@ services:
     App\Util\Api:
         calls:
             - setLogger: [ '@logger' ]
+        public: true
```

Comment:
This is not necessary, Symfony tests have access to a container that includes both public and private services. See `RefreshTest.php` comment